README.md

```diff
@@ -49,44 +49,44 @@ git clone https://github.com/kubeslice/netops.git
 cd netops
 ```
 
-2. Adjust the `VERSION` variable in the Makefile to change the docker tag to be built.
-Image is set as `docker.io/aveshasystems/netops:$(VERSION)` in the Makefile. Change this if required.
+2. Edit the `VERSION` variable in the Makefile to change the docker tag to be built.
+Image is set as `docker.io/aveshasystems/netops:$(VERSION)` in the Makefile. Modiy this if required.
 
 ```bash
 make docker-build
 ```
 
-### Running Locally on Kind Clusters
+### Run Locally on a Kind Clusters
 
 1. You can load the netops image on your kind cluster using the following command:
 
-```bash
-kind load docker-image my-custom-image:unique-tag --name clustername
-```
+   ```bash
+   kind load docker-image my-custom-image:unique-tag --name clustername
+   ```
 
-Example
+  Example
 
-```console
-kind load docker-image aveshasystems/netops:1.2.1 --name kind
-```
+  ```console
+  kind load docker-image aveshasystems/netops:1.2.1 --name kind
+  ```
 
-2. Check the loaded image in the cluster. Modify node name if required.
+2. Check the loaded image in the cluster. Modify the node name if required.
 
-```console
-docker exec -it <node-name> crictl images
-```
+   ```console
+   docker exec -it <node-name> crictl images
+   ```
 
-Example
+   Example
 
-```console
-docker exec -it kind-control-plane crictl images
-```
+   ```console
+   docker exec -it kind-control-plane crictl images
+   ```
 
 
-### Deploy in a Cluster
+### Deploy on a Cluster
 
-Update the chart values file called `yourvaluesfile.yaml` that you have previously created.
-Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-worker/values.yaml) to create `yourvaluesfiel.yaml` and update the netop image subsection to use the local image.
+Update a chart values file called `yourvaluesfile.yaml` that you have previously created.
```

Comment:
`the` is correct  as the file with the same name existed and we are just asking to update that file.